pyproject.toml

```diff
@@ -12,30 +12,30 @@ documentation = "https://grpc-interceptor.readthedocs.io"
 
 [tool.poetry.dependencies]
 python = "^3.6"
-grpcio = "^1.8.0"
-protobuf = {version = ">=3.6.0", optional = true}
+grpcio = "^1.45.0"
```

Comment:
I'd like to avoid bumping dependency constraints as much as possible. That way if users have pinned or capped their versions of `grpcio` for some reason, they can still get updates to `grpc-interceptor`. What was the issue with Python 3.10?

Reply:
@d5h mypy would just not install under python 3.10 and I was getting [this error](https://stackoverflow.com/questions/69528842/an-error-while-trying-to-execute-tests-on-python-3-10-with-pytest) while running pytest under it. I could have just updated these dev dependancies, but I also noticed that while you specify grpcio "^1.8.0", the poetry.lock actually has "1.43.0" which means you are installing and testing against that.
https://github.com/d5h-foss/grpc-interceptor/blob/master/poetry.lock#L968

Given that you are testing against 1.43.0, and there are no guarantees that tests pass against 1.8.0, shouldn't the version of these be bumped up anyway?

In theory, according to semantic versioning, 1.43.0 should be compatible with 1.8.0.